homeworks/franktoth/3_conditionals_loops/flat_finder.py

```diff
@@ -0,0 +1,27 @@
+
+"""Sarah-nak kell segítened a lakáskeresésben, a következőket tudjuk:
+- Nagyon szereti New York-ot és San Fransisco-t, bármelyik városban kivenne egy lakást, ha az albérlet ára kevesebb mint 4000 USD havonta.
+- Gyűlöli Washington-t, és semmi pénzért nem lakna ott
+- Annyira imádja Chicago-t, hogy még a pénz sem akadály, bármit megadna azért hogy ott lakhasson
+- Ha bármilyen más helyről van szó, 3000 USD vagy ez alatti havi lakbér ellenében költözne oda.
+Írj egy programot, amely bekéri a felhasználótól a várost és a lakbér árát. Ezután a fentiek alapján printeld ki egy f-string használatával hogy az adott feltételek (város és albérlet ára) mellett be tudna e költözni az adott helyre.
+"""
+
+
+city = input("Enter the city: ")
+rent = int(input("Enter the monthly rent in USD: "))
+
+if city == "New York" or city == "San Francisco":
```

Comment:
nincs baj az or-ral és a két city vizsgálattal, csak mutatok egy másik megoldást is:
`if city in ["New York", "San Francisco"]: `

Ha több város tartozna ebbe a kategóriába akkor érdemes ezeket eltárolni egy változóban, pl:
`loved_cities = ["New York", "San Francisco", "Boston", "San Jose"]`
aztán csak:
`if city in loved_cities:`

Ehhez a feladathoz tökéletes a mostani megoldás.